specs/v1.yaml

```diff
@@ -1862,7 +1862,7 @@ components:
                 type: string
                 description: |-
                   The identifier of the donor's DAFpay Workflow Session.
-                  See [Integrating Connect](/integrating-connect) for how to get this value from the DAFpay client-side SDK.
+                  See [Capturing Grant Intents](/guides/dafpay/integrating-da-fpay/integration#capturing-grant-intents) for how to get this value from the DAFpay `CHARIOT_SUCCESS` event.
```

Comment:
this might be outside scope of your PR but why is there an extra dash between "da-fpay" ? can we fix that?